test/server/cards/03_TWI/units/PloKoonKohtoyah.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Plo Koon Kohtoyah', function () {
+    integration(function (contextRef) {
+        describe('Plo Koon Kohtoyah\'s ability', function () {
+            it('should ambush attack for 6', function () {
```

Comment:
These are great! For all coordinate tests we also want to test that the ability turns off correctly just to validate that coordinate is set up correctly in the card implementation. Can you repeat this test case with coordinate off?

Reply:
Testing Turning On/Off in the same phase is not easy because Plo koon is exhausted and can't attack a second time.
Not sure yet how to achieve a test containing several phases.

Reply:
you can do context.unit.exhausted = false to un-exhaust unit